src/modules/nfts/components/wizard/MintInProgress.tsx

```diff
@@ -0,0 +1,334 @@
+      if (transactionStep === TransactionStep.META_DATA_UPLOADING) {
+        attemptMetaDataUpload();
+      } else if (transactionStep === TransactionStep.SIGNING) {
+        attemptHolaplexSign();
+      }
+    }
+  }, [
+    isActive,
+    showErrors,
+    nftValue,
+    transactionStep,
+    attemptHolaplexSign,
+    attemptMetaDataUpload,
+    index,
+  ]);
+
+  if (!nftValue) {
+    return null;
+  }
+
+  return (
+    <NavContainer
+      title={`Minting ${index + 1} of ${images.length}`}
+      previousStep={previousStep}
+      goToStep={goToStep}
+    >
+      <Row>
+        <Col style={{ marginRight: 224 }}>
+          <Row>
+            <Paragraph style={{ fontWeight: 900 }}>Progress</Paragraph>
+          </Row>
+          <Row style={{ marginTop: 37 }}>
+            <Col style={{ minWidth: 237 }}>
+              <MintStep
+                text="Uploading metadata"
+                isActive={transactionStep === TransactionStep.META_DATA_UPLOADING}
+                isDone={transactionStep > TransactionStep.META_DATA_UPLOADING}
+                failed={transactionStep === TransactionStep.META_DATA_UPLOAD_FAILED}
+              />
+              <MintStep
+                text="Approving transaction"
+                isActive={transactionStep === TransactionStep.APPROVING}
+                isDone={transactionStep > TransactionStep.APPROVING}
+                failed={transactionStep === TransactionStep.APPROVAL_FAILED}
+              />
+              <MintStep
+                text="Sending transaction to Solana"
+                isActive={transactionStep === TransactionStep.SENDING}
+                isDone={transactionStep > TransactionStep.SENDING}
+                failed={transactionStep === TransactionStep.SENDING_FAILED}
+              />
+              <MintStep
+                text="Waiting for final confirmation"
+                isActive={transactionStep === TransactionStep.FINALIZING}
+                isDone={transactionStep > TransactionStep.FINALIZING}
+              />
+              <MintStep
+                text="Signing Holaplex as co-creator"
+                isActive={transactionStep === TransactionStep.SIGNING}
+                isDone={transactionStep === TransactionStep.SUCCESS}
+                failed={transactionStep === TransactionStep.SENDING_FAILED}
+              />
+            </Col>
+          </Row>
+          {showErrors && (
+            <>
+              <Row style={{ marginTop: 144 }}>
+                <Paragraph style={{ fontSize: 14, color: '#D24040' }}>
+                  {/* TODO: Show error */}
+                  Mint #{index + 1} failed.
+                </Paragraph>
+              </Row>
+              <Row>
+                <Space size={8}>
+                  {transactionStep !== TransactionStep.SIGNING_FAILED && (
+                    <Button
+                      type="default"
+                      onClick={handleNext}
+                      style={{ background: 'rgba(53, 53, 53, 1)' }}
```

Comment:
Prop want to use a different button type if you want an alternate skin for it or at the least styled component.